### PR TITLE
Formatted Readme

### PR DESCRIPTION
**Pull request by CoMoFo**
Added formatted readme as per mockup with it's own scss page.

### CODE REVIEW

File: src/containers/transaction/Transaction.js [hunk 1]
```diff
@@ -32,19 +32,15 @@ const generateTransfersArr = async transaction => {
           const amount = notification.state.value.find(
```
Comment (comountainclimber) [inline src/containers/transaction/Transaction.js]: I dont think this file should actually have any changes in it for this task... Try pulling the latest of `master`

Comment (CoMoFo) [reply src/containers/transaction/Transaction.js]: Should be good now, let me know if I missed anything else!

**Comment by comountainclimber** [comment]
@CoMoFo awesome job! This looks really great and I am super excited that you are contributing to our project. I noticed a few issues that I think we should resolve before merging and deploying your work:

1.) At lower resolutions (375px) the content becomes illegible... I know we dont have mocks for mobile but can you polish up the layout on smaller screens
Image from iOS (5)

2.) 
Group

**A** - It looks like the mock has more vertical padding than the current implementation. Also there is more padding bellow the "Prereqs" header.

**B** - Seems like the line height is a little off on these bullets can we align the vertical spacing to that of the mocks

**C** - I think we need a little more margin on the "numbered sections" divs

**Comment by comountainclimber** [comment]
@CoMoFo let me know if you need further assistance or clarification

**Pull request merged**